worker.go

```diff
@@ -23,6 +25,7 @@ type BridgeWorker struct {
 	// workerChan is used to receive and process job
 	workerChan chan JobHandler
 	listeners  map[string]Listener
+	stopCh     chan struct{}
 	isClose    int32
```

Comment:
Looks like we change from using atomic variable `isClose` to channel `stopCh`, so I think we need to cleanup unused variables and functions `IsClose`